lib/components/ActionTrigger/ActionTriggerButton.tsx

```diff
@@ -35,29 +33,13 @@ export interface ActionTriggerButtonProps {
     attr?: ActionTriggerButtonAttributes & ActionTriggerAttributes;    
 }
 
-const StyledButton = styled(Attr.button)`
```

Comment:
why were we been using this before?

Reply:
this is styled components stuff... this was there for doing white labeling.... now with css custom properties we can do all of that without relying on this package, just by modifying the custom properties